### ISSUE TITLE
RKE2 with cilium pods and cilium have CrashLoopBackOff state

### ISSUE THREAD

**Issue by ByAziX** [cilium/cilium#23265]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

hello, I created a kubernetes RKE2 cluster as I saw that RKE2 already implement cilium, I just added it to the configuration file during my kubernetes deployment. However, I have my cilium pods in CrashLoopBackOff state. I don't understand why here are some screenshots:

[kubectl get pods -A](https://i.stack.imgur.com/oWfCT.png)

[cilium sysdump](https://i.stack.imgur.com/fur0D.png)

kubectl logs cilium-gj9zf -n kube-system

I tried to install cilium without going through the default configuration of rke2 it did not work

I followed this tuto to do it :

https://gitlab.com/monachus/channel/-/tree/master/resources/2021-09-07-ha-rke2-kube-vip-rancher

thanks you =)

### Cilium Version

cilium-cli: v0.12.12 compiled with go1.19.4 on linux/amd64
cilium image (default): v1.12.5
cilium image (stable): v1.12.5
cilium image (running): unknown. Unable to obtain cilium version, no cilium pods found in namespace "kube-system"

### Kernel Version

Linux test 5.15.0-1025-kvm #30-Ubuntu SMP Thu Dec 1 22:50:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

### Kubernetes Version

root@test:~# kubectl version --output=yaml
clientVersion:
  buildDate: "2023-01-04T18:07:51Z"
  compiler: gc
  gitCommit: 9710807c82740b9799453677c977758becf0acbb
  gitTreeState: clean
  gitVersion: v1.24.9+rke2r2
  goVersion: go1.18.9b7
  major: "1"
  minor: "24"
  platform: linux/amd64
kustomizeVersion: v4.5.4
serverVersion:
  buildDate: "2023-01-04T18:07:51Z"
  compiler: gc
  gitCommit: 9710807c82740b9799453677c977758becf0acbb
  gitTreeState: clean
  gitVersion: v1.24.9+rke2r2
  goVersion: go1.18.9b7
  major: "1"
  minor: "24"
  platform: linux/amd64

### Sysdump

[cilium_sysdump.zip](https://github.com/cilium/cilium/files/10483939/cilium_sysdump.zip)

### Relevant log output

```shell
root@test:/etc/NetworkManager/dispatcher.d# kubectl logs cilium-gj9zf -n kube-system 
Defaulted container "cilium-agent" out of: cilium-agent, sysdump-1674485365 (ephem), sysdump-1674486593 (ephem), install-portmap-cni-plugin (init), mount-cgroup (init), apply-sysctl-overwrites (init), mount-bpf-fs (init), clean-cilium-state (init)
level=info msg="Started gops server" address="127.0.0.1:9890" subsys=daemon
level=info msg="Memory available for map entries (0.003% of 4130906112B): 10327265B" subsys=config
level=info msg="option bpf-ct-global-tcp-max set by dynamic sizing to 131072" subsys=config
level=info msg="option bpf-ct-global-any-max set by dynamic sizing to 65536" subsys=config
level=info msg="option bpf-nat-global-max set by dynamic sizing to 131072" subsys=config
level=info msg="option bpf-neigh-global-max set by dynamic sizing to 131072" subsys=config
level=info msg="option bpf-sock-rev-map-max set by dynamic sizing to 65536" subsys=config
level=info msg="  --agent-health-port='9879'" subsys=daemon
level=info msg="  --agent-labels=''" subsys=daemon
level=info msg="  --agent-not-ready-taint-key='node.cilium.io/agent-not-ready'" subsys=daemon
level=info msg="  --allocator-list-timeout='3m0s'" subsys=daemon
level=info msg="  --allow-icmp-frag-needed='true'" subsys=daemon
level=info msg="  --allow-localhost='auto'" subsys=daemon
level=info msg="  --annotate-k8s-node='false'" subsys=daemon
level=info msg="  --api-rate-limit=''" subsys=daemon
level=info msg="  --arping-refresh-period='30s'" subsys=daemon
level=info msg="  --auto-create-cilium-node-resource='true'" subsys=daemon
level=info msg="  --auto-direct-node-routes='false'" subsys=daemon
level=info msg="  --bgp-announce-lb-ip='false'" subsys=daemon
level=info msg="  --bgp-announce-pod-cidr='false'" subsys=daemon
level=info msg="  --bgp-config-path='/var/lib/cilium/bgp/config.yaml'" subsys=daemon
level=info msg="  --bpf-ct-global-any-max='262144'" subsys=daemon
level=info msg="  --bpf-ct-global-tcp-max='524288'" subsys=daemon
level=info msg="  --bpf-ct-timeout-regular-any='1m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-regular-tcp='6h0m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-regular-tcp-fin='10s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-regular-tcp-syn='1m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-service-any='1m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-service-tcp='6h0m0s'" subsys=daemon
level=info msg="  --bpf-ct-timeout-service-tcp-grace='1m0s'" subsys=daemon
level=info msg="  --bpf-filter-priority='1'" subsys=daemon
level=info msg="  --bpf-fragments-map-max='8192'" subsys=daemon
level=info msg="  --bpf-lb-acceleration='disabled'" subsys=daemon
level=info msg="  --bpf-lb-affinity-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-algorithm='random'" subsys=daemon
level=info msg="  --bpf-lb-dev-ip-addr-inherit=''" subsys=daemon
level=info msg="  --bpf-lb-dsr-dispatch='opt'" subsys=daemon
level=info msg="  --bpf-lb-dsr-l4-xlate='frontend'" subsys=daemon
level=info msg="  --bpf-lb-external-clusterip='false'" subsys=daemon
level=info msg="  --bpf-lb-maglev-hash-seed='JLfvgnHc2kaSUFaI'" subsys=daemon
level=info msg="  --bpf-lb-maglev-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-maglev-table-size='16381'" subsys=daemon
level=info msg="  --bpf-lb-map-max='65536'" subsys=daemon
level=info msg="  --bpf-lb-mode='snat'" subsys=daemon
level=info msg="  --bpf-lb-rev-nat-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-rss-ipv4-src-cidr=''" subsys=daemon
level=info msg="  --bpf-lb-rss-ipv6-src-cidr=''" subsys=daemon
level=info msg="  --bpf-lb-service-backend-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-service-map-max='0'" subsys=daemon
level=info msg="  --bpf-lb-sock='false'" subsys=daemon
level=info msg="  --bpf-lb-sock-hostns-only='false'" subsys=daemon
level=info msg="  --bpf-lb-source-range-map-max='0'" subsys=daemon
level=info msg="  --bpf-map-dynamic-size-ratio='0.0025'" subsys=daemon
level=info msg="  --bpf-nat-global-max='524288'" subsys=daemon
level=info msg="  --bpf-neigh-global-max='524288'" subsys=daemon
level=info msg="  --bpf-policy-map-max='16384'" subsys=daemon
level=info msg="  --bpf-root='/sys/fs/bpf'" subsys=daemon
level=info msg="  --bpf-sock-rev-map-max='262144'" subsys=daemon
level=info msg="  --bypass-ip-availability-upon-restore='false'" subsys=daemon
level=info msg="  --certificates-directory='/var/run/cilium/certs'" subsys=daemon
level=info msg="  --cflags=''" subsys=daemon
level=info msg="  --cgroup-root='/run/cilium/cgroupv2'" subsys=daemon
level=info msg="  --cluster-health-port='4240'" subsys=daemon
level=info msg="  --cluster-id='0'" subsys=daemon
level=info msg="  --cluster-name='default'" subsys=daemon
level=info msg="  --clustermesh-config='/var/lib/cilium/clustermesh/'" subsys=daemon
level=info msg="  --cmdref=''" subsys=daemon
level=info msg="  --cni-chaining-mode='portmap'" subsys=daemon
level=info msg="  --config=''" subsys=daemon
level=info msg="  --config-dir='/tmp/cilium/config-map'" subsys=daemon
level=info msg="  --conntrack-gc-interval='0s'" subsys=daemon
level=info msg="  --crd-wait-timeout='5m0s'" subsys=daemon
level=info msg="  --datapath-mode='veth'" subsys=daemon
level=info msg="  --debug='false'" subsys=daemon
level=info msg="  --debug-verbose=''" subsys=daemon
level=info msg="  --derive-masquerade-ip-addr-from-device=''" subsys=daemon
level=info msg="  --devices=''" subsys=daemon
level=info msg="  --direct-routing-device=''" subsys=daemon
level=info msg="  --disable-cnp-status-updates='true'" subsys=daemon
level=info msg="  --disable-conntrack='false'" subsys=daemon
level=info msg="  --disable-endpoint-crd='false'" subsys=daemon
level=info msg="  --disable-envoy-version-check='false'" subsys=daemon
level=info msg="  --disable-iptables-feeder-rules=''" subsys=daemon
level=info msg="  --dns-max-ips-per-restored-rule='1000'" subsys=daemon
level=info msg="  --dns-policy-unload-on-shutdown='false'" subsys=daemon
level=info msg="  --dnsproxy-concurrency-limit='0'" subsys=daemon
level=info msg="  --dnsproxy-concurrency-processing-grace-period='0s'" subsys=daemon
level=info msg="  --egress-masquerade-interfaces=''" subsys=daemon
level=info msg="  --egress-multi-home-ip-rule-compat='false'" subsys=daemon
level=info msg="  --enable-auto-protect-node-port-range='true'" subsys=daemon
level=info msg="  --enable-bandwidth-manager='false'" subsys=daemon
level=info msg="  --enable-bbr='false'" subsys=daemon
level=info msg="  --enable-bgp-control-plane='false'" subsys=daemon
level=info msg="  --enable-bpf-clock-probe='true'" subsys=daemon
level=info msg="  --enable-bpf-masquerade='false'" subsys=daemon
level=info msg="  --enable-bpf-tproxy='false'" subsys=daemon
level=info msg="  --enable-cilium-endpoint-slice='false'" subsys=daemon
level=info msg="  --enable-custom-calls='false'" subsys=daemon
level=info msg="  --enable-endpoint-health-checking='true'" subsys=daemon
level=info msg="  --enable-endpoint-routes='false'" subsys=daemon
level=info msg="  --enable-envoy-config='false'" subsys=daemon
level=info msg="  --enable-external-ips='true'" subsys=daemon
level=info msg="  --enable-health-check-nodeport='true'" subsys=daemon
level=info msg="  --enable-health-checking='true'" subsys=daemon
level=info msg="  --enable-host-firewall='false'" subsys=daemon
level=info msg="  --enable-host-legacy-routing='false'" subsys=daemon
level=info msg="  --enable-host-port='true'" subsys=daemon
level=info msg="  --enable-host-reachable-services='false'" subsys=daemon
level=info msg="  --enable-hubble='false'" subsys=daemon
level=info msg="  --enable-hubble-recorder-api='true'" subsys=daemon
level=info msg="  --enable-icmp-rules='true'" subsys=daemon
level=info msg="  --enable-identity-mark='true'" subsys=daemon
level=info msg="  --enable-ip-masq-agent='false'" subsys=daemon
level=info msg="  --enable-ipsec='false'" subsys=daemon
level=info msg="  --enable-ipv4='true'" subsys=daemon
level=info msg="  --enable-ipv4-egress-gateway='false'" subsys=daemon
level=info msg="  --enable-ipv4-fragment-tracking='true'" subsys=daemon
level=info msg="  --enable-ipv4-masquerade='true'" subsys=daemon
level=info msg="  --enable-ipv6='false'" subsys=daemon
level=info msg="  --enable-ipv6-masquerade='true'" subsys=daemon
level=info msg="  --enable-ipv6-ndp='false'" subsys=daemon
level=info msg="  --enable-k8s-api-discovery='false'" subsys=daemon
level=info msg="  --enable-k8s-endpoint-slice='true'" subsys=daemon
level=info msg="  --enable-k8s-event-handover='false'" subsys=daemon
level=info msg="  --enable-k8s-terminating-endpoint='true'" subsys=daemon
level=info msg="  --enable-l2-neigh-discovery='true'" subsys=daemon
level=info msg="  --enable-l7-proxy='true'" subsys=daemon
level=info msg="  --enable-local-node-route='true'" subsys=daemon
level=info msg="  --enable-local-redirect-policy='false'" subsys=daemon
level=info msg="  --enable-mke='false'" subsys=daemon
level=info msg="  --enable-monitor='true'" subsys=daemon
level=info msg="  --enable-node-port='false'" subsys=daemon
level=info msg="  --enable-policy='default'" subsys=daemon
level=info msg="  --enable-recorder='false'" subsys=daemon
level=info msg="  --enable-remote-node-identity='true'" subsys=daemon
level=info msg="  --enable-runtime-device-detection='false'" subsys=daemon
level=info msg="  --enable-selective-regeneration='true'" subsys=daemon
level=info msg="  --enable-service-topology='false'" subsys=daemon
level=info msg="  --enable-session-affinity='false'" subsys=daemon
level=info msg="  --enable-svc-source-range-check='true'" subsys=daemon
level=info msg="  --enable-tracing='false'" subsys=daemon
level=info msg="  --enable-unreachable-routes='false'" subsys=daemon
level=info msg="  --enable-vtep='false'" subsys=daemon
level=info msg="  --enable-well-known-identities='false'" subsys=daemon
level=info msg="  --enable-wireguard='false'" subsys=daemon
level=info msg="  --enable-wireguard-userspace-fallback='false'" subsys=daemon
level=info msg="  --enable-xdp-prefilter='false'" subsys=daemon
level=info msg="  --enable-xt-socket-fallback='true'" subsys=daemon
level=info msg="  --encrypt-interface=''" subsys=daemon
level=info msg="  --encrypt-node='false'" subsys=daemon
level=info msg="  --endpoint-gc-interval='5m0s'" subsys=daemon
level=info msg="  --endpoint-interface-name-prefix=''" subsys=daemon
level=info msg="  --endpoint-queue-size='25'" subsys=daemon
level=info msg="  --endpoint-status=''" subsys=daemon
level=info msg="  --envoy-config-timeout='2m0s'" subsys=daemon
level=info msg="  --envoy-log=''" subsys=daemon
level=info msg="  --exclude-local-address=''" subsys=daemon
level=info msg="  --fixed-identity-mapping=''" subsys=daemon
level=info msg="  --force-local-policy-eval-at-source='true'" subsys=daemon
level=info msg="  --fqdn-regex-compile-lru-size='1024'" subsys=daemon
level=info msg="  --gops-port='9890'" subsys=daemon
level=info msg="  --host-reachable-services-protos='tcp,udp'" subsys=daemon
level=info msg="  --http-403-msg=''" subsys=daemon
level=info msg="  --http-idle-timeout='0'" subsys=daemon
level=info msg="  --http-max-grpc-timeout='0'" subsys=daemon
level=info msg="  --http-normalize-path='true'" subsys=daemon
level=info msg="  --http-request-timeout='3600'" subsys=daemon
level=info msg="  --http-retry-count='3'" subsys=daemon
level=info msg="  --http-retry-timeout='0'" subsys=daemon
level=info msg="  --hubble-disable-tls='false'" subsys=daemon
level=info msg="  --hubble-event-buffer-capacity='4095'" subsys=daemon
level=info msg="  --hubble-event-queue-size='0'" subsys=daemon
level=info msg="  --hubble-export-file-compress='false'" subsys=daemon
level=info msg="  --hubble-export-file-max-backups='5'" subsys=daemon
level=info msg="  --hubble-export-file-max-size-mb='10'" subsys=daemon
level=info msg="  --hubble-export-file-path=''" subsys=daemon
level=info msg="  --hubble-listen-address=''" subsys=daemon
level=info msg="  --hubble-metrics=''" subsys=daemon
level=info msg="  --hubble-metrics-server=''" subsys=daemon
level=info msg="  --hubble-recorder-sink-queue-size='1024'" subsys=daemon
level=info msg="  --hubble-recorder-storage-path='/var/run/cilium/pcaps'" subsys=daemon
level=info msg="  --hubble-socket-path='/var/run/cilium/hubble.sock'" subsys=daemon
level=info msg="  --hubble-tls-cert-file=''" subsys=daemon
level=info msg="  --hubble-tls-client-ca-files=''" subsys=daemon
level=info msg="  --hubble-tls-key-file=''" subsys=daemon
level=info msg="  --identity-allocation-mode='crd'" subsys=daemon
level=info msg="  --identity-change-grace-period='5s'" subsys=daemon
level=info msg="  --identity-restore-grace-period='10m0s'" subsys=daemon
level=info msg="  --install-egress-gateway-routes='false'" subsys=daemon
level=info msg="  --install-iptables-rules='true'" subsys=daemon
level=info msg="  --install-no-conntrack-iptables-rules='false'" subsys=daemon
level=info msg="  --ip-allocation-timeout='2m0s'" subsys=daemon
level=info msg="  --ip-masq-agent-config-path='/etc/config/ip-masq-agent'" subsys=daemon
level=info msg="  --ipam='kubernetes'" subsys=daemon
level=info msg="  --ipsec-key-file=''" subsys=daemon
level=info msg="  --iptables-lock-timeout='5s'" subsys=daemon
level=info msg="  --iptables-random-fully='false'" subsys=daemon
level=info msg="  --ipv4-native-routing-cidr=''" subsys=daemon
level=info msg="  --ipv4-node='auto'" subsys=daemon
level=info msg="  --ipv4-pod-subnets=''" subsys=daemon
level=info msg="  --ipv4-range='auto'" subsys=daemon
level=info msg="  --ipv4-service-loopback-address='169.254.42.1'" subsys=daemon
level=info msg="  --ipv4-service-range='auto'" subsys=daemon
level=info msg="  --ipv6-cluster-alloc-cidr='f00d::/64'" subsys=daemon
level=info msg="  --ipv6-mcast-device=''" subsys=daemon
level=info msg="  --ipv6-native-routing-cidr=''" subsys=daemon
level=info msg="  --ipv6-node='auto'" subsys=daemon
level=info msg="  --ipv6-pod-subnets=''" subsys=daemon
level=info msg="  --ipv6-range='auto'" subsys=daemon
level=info msg="  --ipv6-service-range='auto'" subsys=daemon
level=info msg="  --join-cluster='false'" subsys=daemon
level=info msg="  --k8s-api-server=''" subsys=daemon
level=info msg="  --k8s-heartbeat-timeout='30s'" subsys=daemon
level=info msg="  --k8s-kubeconfig-path=''" subsys=daemon
level=info msg="  --k8s-namespace='kube-system'" subsys=daemon
level=info msg="  --k8s-require-ipv4-pod-cidr='false'" subsys=daemon
level=info msg="  --k8s-require-ipv6-pod-cidr='false'" subsys=daemon
level=info msg="  --k8s-service-cache-size='128'" subsys=daemon
level=info msg="  --k8s-service-proxy-name=''" subsys=daemon
level=info msg="  --k8s-sync-timeout='3m0s'" subsys=daemon
level=info msg="  --k8s-watcher-endpoint-selector='metadata.name!=kube-scheduler,metadata.name!=kube-controller-manager,metadata.name!=etcd-operator,metadata.name!=gcp-controller-manager'" subsys=daemon
level=info msg="  --keep-config='false'" subsys=daemon
level=info msg="  --kube-proxy-replacement='disabled'" subsys=daemon
level=info msg="  --kube-proxy-replacement-healthz-bind-address=''" subsys=daemon
level=info msg="  --kvstore=''" subsys=daemon
level=info msg="  --kvstore-connectivity-timeout='2m0s'" subsys=daemon
level=info msg="  --kvstore-lease-ttl='15m0s'" subsys=daemon
level=info msg="  --kvstore-max-consecutive-quorum-errors='2'" subsys=daemon
level=info msg="  --kvstore-opt=''" subsys=daemon
level=info msg="  --kvstore-periodic-sync='5m0s'" subsys=daemon
level=info msg="  --label-prefix-file=''" subsys=daemon
level=info msg="  --labels=''" subsys=daemon
level=info msg="  --lib-dir='/var/lib/cilium'" subsys=daemon
level=info msg="  --local-max-addr-scope='252'" subsys=daemon
level=info msg="  --local-router-ipv4=''" subsys=daemon
level=info msg="  --local-router-ipv6=''" subsys=daemon
level=info msg="  --log-driver=''" subsys=daemon
level=info msg="  --log-opt=''" subsys=daemon
level=info msg="  --log-system-load='false'" subsys=daemon
level=info msg="  --max-controller-interval='0'" subsys=daemon
level=info msg="  --metrics=''" subsys=daemon
level=info msg="  --mke-cgroup-mount=''" subsys=daemon
level=info msg="  --monitor-aggregation='medium'" subsys=daemon
level=info msg="  --monitor-aggregation-flags='all'" subsys=daemon
level=info msg="  --monitor-aggregation-interval='5s'" subsys=daemon
level=info msg="  --monitor-queue-size='0'" subsys=daemon
level=info msg="  --mtu='0'" subsys=daemon
level=info msg="  --node-port-acceleration='disabled'" subsys=daemon
level=info msg="  --node-port-algorithm='random'" subsys=daemon
level=info msg="  --node-port-bind-protection='true'" subsys=daemon
level=info msg="  --node-port-mode='snat'" subsys=daemon
level=info msg="  --node-port-range='30000,32767'" subsys=daemon
level=info msg="  --policy-audit-mode='false'" subsys=daemon
level=info msg="  --policy-queue-size='100'" subsys=daemon
level=info msg="  --policy-trigger-interval='1s'" subsys=daemon
level=info msg="  --pprof='false'" subsys=daemon
level=info msg="  --pprof-port='6060'" subsys=daemon
level=info msg="  --preallocate-bpf-maps='false'" subsys=daemon
level=info msg="  --prepend-iptables-chains='true'" subsys=daemon
level=info msg="  --procfs='/host/proc'" subsys=daemon
level=info msg="  --prometheus-serve-addr=':9962'" subsys=daemon
level=info msg="  --proxy-connect-timeout='1'" subsys=daemon
level=info msg="  --proxy-gid='1337'" subsys=daemon
level=info msg="  --proxy-max-connection-duration-seconds='0'" subsys=daemon
level=info msg="  --proxy-max-requests-per-connection='0'" subsys=daemon
level=info msg="  --proxy-prometheus-port='9964'" subsys=daemon
level=info msg="  --read-cni-conf=''" subsys=daemon
level=info msg="  --restore='true'" subsys=daemon
level=info msg="  --route-metric='0'" subsys=daemon
level=info msg="  --sidecar-istio-proxy-image='cilium/istio_proxy'" subsys=daemon
level=info msg="  --single-cluster-route='false'" subsys=daemon
level=info msg="  --socket-path='/var/run/cilium/cilium.sock'" subsys=daemon
level=info msg="  --sockops-enable='false'" subsys=daemon
level=info msg="  --state-dir='/var/run/cilium'" subsys=daemon
level=info msg="  --tofqdns-dns-reject-response-code='refused'" subsys=daemon
level=info msg="  --tofqdns-enable-dns-compression='true'" subsys=daemon
level=info msg="  --tofqdns-endpoint-max-ip-per-hostname='50'" subsys=daemon
level=info msg="  --tofqdns-idle-connection-grace-period='0s'" subsys=daemon
level=info msg="  --tofqdns-max-deferred-connection-deletes='10000'" subsys=daemon
level=info msg="  --tofqdns-min-ttl='3600'" subsys=daemon
level=info msg="  --tofqdns-pre-cache=''" subsys=daemon
level=info msg="  --tofqdns-proxy-port='0'" subsys=daemon
level=info msg="  --tofqdns-proxy-response-max-delay='100ms'" subsys=daemon
level=info msg="  --trace-payloadlen='128'" subsys=daemon
level=info msg="  --tunnel='vxlan'" subsys=daemon
level=info msg="  --tunnel-port='0'" subsys=daemon
level=info msg="  --version='false'" subsys=daemon
level=info msg="  --vlan-bpf-bypass=''" subsys=daemon
level=info msg="  --vtep-cidr=''" subsys=daemon
level=info msg="  --vtep-endpoint=''" subsys=daemon
level=info msg="  --vtep-mac=''" subsys=daemon
level=info msg="  --vtep-mask=''" subsys=daemon
level=info msg="  --write-cni-conf-when-ready=''" subsys=daemon
level=info msg="     _ _ _" subsys=daemon
level=info msg=" ___|_| |_|_ _ _____" subsys=daemon
level=info msg="|  _| | | | | |     |" subsys=daemon
level=info msg="|___|_|_|_|___|_|_|_|" subsys=daemon
level=info msg="Cilium 1.12.4 6eaecaf 2022-11-16T05:45:01+00:00 go version go1.18.8 linux/amd64" subsys=daemon
level=info msg="cilium-envoy  version: 48cbb2f88edca93ba80d1135dbb067a827b8adc2/1.21.5/Distribution/RELEASE/BoringSSL" subsys=daemon
level=info msg="clang (10.0.0) and kernel (5.15.0) versions: OK!" subsys=linux-datapath
level=info msg="linking environment: OK!" subsys=linux-datapath
level=info msg="Detected mounted BPF filesystem at /sys/fs/bpf" subsys=bpf
level=info msg="Mounted cgroupv2 filesystem at /run/cilium/cgroupv2" subsys=cgroups
level=info msg="Parsing base label prefixes from default label list" subsys=labels-filter
level=info msg="Parsing additional label prefixes from user inputs: []" subsys=labels-filter
level=info msg="Final label prefixes to be used for identity evaluation:" subsys=labels-filter
level=info msg=" - reserved:.*" subsys=labels-filter
level=info msg=" - :io\\.kubernetes\\.pod\\.namespace" subsys=labels-filter
level=info msg=" - :io\\.cilium\\.k8s\\.namespace\\.labels" subsys=labels-filter
level=info msg=" - :app\\.kubernetes\\.io" subsys=labels-filter
level=info msg=" - !:io\\.kubernetes" subsys=labels-filter
level=info msg=" - !:kubernetes\\.io" subsys=labels-filter
level=info msg=" - !:.*beta\\.kubernetes\\.io" subsys=labels-filter
level=info msg=" - !:k8s\\.io" subsys=labels-filter
level=info msg=" - !:pod-template-generation" subsys=labels-filter
level=info msg=" - !:pod-template-hash" subsys=labels-filter
level=info msg=" - !:controller-revision-hash" subsys=labels-filter
level=info msg=" - !:annotation.*" subsys=labels-filter
level=info msg=" - !:etcd_node" subsys=labels-filter
level=info msg="Auto-disabling \"enable-bpf-clock-probe\" feature since KERNEL_HZ cannot be determined" error="Cannot probe CONFIG_HZ" subsys=daemon
level=info msg="Using autogenerated IPv4 allocation range" subsys=node v4Prefix=10.59.0.0/16
level=info msg="Initializing daemon" subsys=daemon
level=info msg="Establishing connection to apiserver" host="https://10.43.0.1:443" subsys=k8s
level=info msg="Connected to apiserver" subsys=k8s
level=info msg="Auto-disabling \"enable-node-port\", \"enable-external-ips\", \"bpf-lb-sock\", \"enable-host-port\" features and falling back to \"enable-host-legacy-routing\"" subsys=daemon
level=info msg="Inheriting MTU from external network interface" device=eth0 ipAddr=10.10.10.59 mtu=1500 subsys=mtu
level=info msg="Envoy: Starting xDS gRPC server listening on /var/run/cilium/xds.sock" subsys=envoy-manager
level=info msg="Restored backends from maps" failedBackends=0 restoredBackends=1 subsys=service
level=info msg="Restored services from maps" failedServices=0 restoredServices=2 subsys=service
level=info msg="Reading old endpoints..." subsys=daemon
level=info msg="No old endpoints found." subsys=daemon
level=info msg="Waiting until all Cilium CRDs are available" subsys=k8s
level=info msg="All Cilium CRDs have been found and are available" subsys=k8s
level=info msg="Retrieved node information from kubernetes node" nodeName=test subsys=k8s
level=info msg="Received own node information from API server" ipAddr.ipv4=10.10.10.59 ipAddr.ipv6="<nil>" k8sNodeIP=10.10.10.59 labels="map[beta.kubernetes.io/arch:amd64 beta.kubernetes.io/instance-type:rke2 beta.kubernetes.io/os:linux egress.rke2.io/cluster:true kubernetes.io/arch:amd64 kubernetes.io/hostname:test kubernetes.io/os:linux node-role.kubernetes.io/control-plane:true node-role.kubernetes.io/etcd:true node-role.kubernetes.io/master:true node.kubernetes.io/instance-type:rke2]" nodeName=test subsys=k8s v4Prefix=10.42.0.0/24 v6Prefix="<nil>"
level=info msg="k8s mode: Allowing localhost to reach local endpoints" subsys=daemon
level=info msg="Detected devices" devices="[]" subsys=linux-datapath
level=info msg="Enabling k8s event listener" subsys=k8s-watcher
level=info msg="Removing stale endpoint interfaces" subsys=daemon
level=info msg="Skipping kvstore configuration" subsys=daemon
level=info msg="Restored router address from node_config" file=/var/run/cilium/state/globals/node_config.h ipv4=10.42.0.241 ipv6="<nil>" subsys=node
level=info msg="Initializing node addressing" subsys=daemon
level=info msg="Initializing kubernetes IPAM" subsys=ipam v4Prefix=10.42.0.0/24 v6Prefix="<nil>"
level=info msg="Restoring endpoints..." subsys=daemon
level=info msg="Endpoints restored" failed=0 restored=0 subsys=daemon
level=info msg="Addressing information:" subsys=daemon
level=info msg="  Cluster-Name: default" subsys=daemon
level=info msg="  Cluster-ID: 0" subsys=daemon
level=info msg="  Local node-name: test" subsys=daemon
level=info msg="  Node-IPv6: <nil>" subsys=daemon
level=info msg="  External-Node IPv4: 10.10.10.59" subsys=daemon
level=info msg="  Internal-Node IPv4: 10.42.0.241" subsys=daemon
level=info msg="  IPv4 allocation prefix: 10.42.0.0/24" subsys=daemon
level=info msg="  Loopback IPv4: 169.254.42.1" subsys=daemon
level=info msg="  Local IPv4 addresses:" subsys=daemon
level=info msg="  - 10.10.10.59" subsys=daemon
level=info msg="  - 10.42.0.241" subsys=daemon
level=info msg="Creating or updating CiliumNode resource" node=test subsys=nodediscovery
level=warning msg="Unable to get node resource" error="CiliumNode.cilium \"test\" not found" subsys=nodediscovery
level=info msg="Waiting until local node addressing before starting watchers depending on it" subsys=k8s-watcher
level=info msg="Waiting until all pre-existing resources have been received" subsys=k8s-watcher
level=info msg="Adding local node to cluster" node="{test default [{InternalIP 10.10.10.59} {CiliumInternalIP 10.42.0.241}] 10.42.0.0/24 [] <nil> [] 10.42.0.151 <nil> <nil> <nil> 0 local 0 map[beta.kubernetes.io/arch:amd64 beta.kubernetes.io/instance-type:rke2 beta.kubernetes.io/os:linux egress.rke2.io/cluster:true kubernetes.io/arch:amd64 kubernetes.io/hostname:test kubernetes.io/os:linux node-role.kubernetes.io/control-plane:true node-role.kubernetes.io/etcd:true node-role.kubernetes.io/master:true node.kubernetes.io/instance-type:rke2] 1 }" subsys=nodediscovery
level=warning msg="Unable to create CiliumNode resource, will retry" error="ciliumnodes.cilium.io \"test\" already exists" subsys=nodediscovery
level=info msg="Initializing identity allocator" subsys=identity-cache
level=info msg="Allocating identities between range" cluster-id=0 max=65535 min=256 subsys=identity-cache
level=info msg="Cluster-ID is not specified, skipping ClusterMesh initialization" subsys=daemon
level=info msg="Setting up BPF datapath" bpfClockSource=ktime bpfInsnSet=v3 subsys=datapath-loader
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.core.bpf_jit_enable sysParamValue=1
level=warning msg="Unable to ensure that BPF JIT compilation is enabled. This can be ignored when Cilium is running inside non-host network namespace (e.g. with kind or minikube)" error="could not open the sysctl file /host/proc/sys/net/core/bpf_jit_enable: open /host/proc/sys/net/core/bpf_jit_enable: no such file or directory" subsys=sysctl sysParamName=net.core.bpf_jit_enable sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.all.rp_filter sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.fib_multipath_use_neigh sysParamValue=1
level=warning msg="Failed to sysctl -w" error="could not open the sysctl file /host/proc/sys/net/ipv4/fib_multipath_use_neigh: open /host/proc/sys/net/ipv4/fib_multipath_use_neigh: no such file or directory" subsys=sysctl sysParamName=net.ipv4.fib_multipath_use_neigh sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=kernel.unprivileged_bpf_disabled sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=kernel.timer_migration sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.forwarding sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.rp_filter sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.accept_local sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.send_redirects sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.forwarding sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.rp_filter sysParamValue=0
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.accept_local sysParamValue=1
level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.send_redirects sysParamValue=0
level=error msg="Command execution failed" cmd="[/var/lib/cilium/bpf/init.sh /var/lib/cilium/bpf /var/run/cilium/state /host/proc/sys/net /sys/class/net 10.42.0.241 <nil> tunnel vxlan 8472 <nil> cilium_host cilium_net 1500 false false false /run/cilium/cgroupv2 /sys/fs/bpf false true v3 2 false true 1]" error="exit status 1" subsys=datapath-loader
level=warning msg="+ set -o pipefail" subsys=datapath-loader
level=warning msg="+ rm /var/run/cilium/state/encap.state" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ DIR=/run/cilium/state/globals" subsys=datapath-loader
level=warning msg="+ case \"${MODE}\" in" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*CILIUM_NET_MAC.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="++ ip link show cilium_net" subsys=datapath-loader
level=warning msg="++ awk '{print $2}'" subsys=datapath-loader
level=warning msg="++ grep ether" subsys=datapath-loader
level=warning msg="+ CILIUM_NET_MAC=f6:de:9f:ca:ac:22" subsys=datapath-loader
level=warning msg="++ mac2array f6:de:9f:ca:ac:22" subsys=datapath-loader
level=warning msg="++ echo '{0xf6,0xde,0x9f,0xca,0xac,0x22}'" subsys=datapath-loader
level=warning msg="+ CILIUM_NET_MAC='{0xf6,0xde,0x9f,0xca,0xac,0x22}'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*CILIUM_NET_MAC.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="+ echo '#ifndef CILIUM_NET_MAC'" subsys=datapath-loader
level=warning msg="+ echo '#define CILIUM_NET_MAC { .addr = {0xf6,0xde,0x9f,0xca,0xac,0x22}}'" subsys=datapath-loader
level=warning msg="+ echo '#endif /* CILIUM_NET_MAC */'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*HOST_IFINDEX.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="++ cat /sys/class/net/cilium_net/ifindex" subsys=datapath-loader
level=warning msg="+ HOST_IDX=99" subsys=datapath-loader
level=warning msg="+ echo '#define HOST_IFINDEX 99'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*HOST_IFINDEX_MAC.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="++ ip link show cilium_host" subsys=datapath-loader
level=warning msg="++ grep ether" subsys=datapath-loader
level=warning msg="++ awk '{print $2}'" subsys=datapath-loader
level=warning msg="+ HOST_MAC=3a:b8:d8:eb:ba:5f" subsys=datapath-loader
level=warning msg="++ mac2array 3a:b8:d8:eb:ba:5f" subsys=datapath-loader
level=warning msg="++ echo '{0x3a,0xb8,0xd8,0xeb,0xba,0x5f}'" subsys=datapath-loader
level=warning msg="+ HOST_MAC='{0x3a,0xb8,0xd8,0xeb,0xba,0x5f}'" subsys=datapath-loader
level=warning msg="+ echo '#define HOST_IFINDEX_MAC { .addr = {0x3a,0xb8,0xd8,0xeb,0xba,0x5f}}'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*CILIUM_IFINDEX.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="++ cat /sys/class/net/cilium_host/ifindex" subsys=datapath-loader
level=warning msg="+ CILIUM_IDX=100" subsys=datapath-loader
level=warning msg="+ echo '#define CILIUM_IFINDEX 100'" subsys=datapath-loader
level=warning msg="++ cat /host/proc/sys/net/ipv4/ip_local_port_range" subsys=datapath-loader
level=warning msg="++ awk '{print $1}'" subsys=datapath-loader
level=warning msg="+ CILIUM_EPHEMERAL_MIN=32768" subsys=datapath-loader
level=warning msg="+ echo '#define EPHEMERAL_MIN 32768'" subsys=datapath-loader
level=warning msg="+ '[' '<nil>' '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ '[' 10.42.0.241 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="++ ip -4 addr show to 10.42.0.241 dev cilium_host" subsys=datapath-loader
level=warning msg="+ '[' -n '100: cilium_host@cilium_net: <BROADCAST,MULTICAST,NOARP,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000" subsys=datapath-loader
level=warning msg="    inet 10.42.0.241/32 scope link cilium_host" subsys=datapath-loader
level=warning msg="       valid_lft forever preferred_lft forever' ']'" subsys=datapath-loader
level=warning msg="+ '[' true = true ']'" subsys=datapath-loader
level=warning msg="+ move_local_rules" subsys=datapath-loader
level=warning msg="+ '[' 10.42.0.241 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ move_local_rules_af 'ip -4'" subsys=datapath-loader
level=warning msg="+ IP='ip -4'" subsys=datapath-loader
level=warning msg="++ ip -4 rule list" subsys=datapath-loader
level=warning msg="+ '[' -z '9:\tfrom all fwmark 0x200/0xf00 lookup 2004" subsys=datapath-loader
level=warning msg="10:\tfrom all fwmark 0xa00/0xf00 lookup 2005" subsys=datapath-loader
level=warning msg="100:\tfrom all lookup local" subsys=datapath-loader
level=warning msg="32766:\tfrom all lookup main" subsys=datapath-loader
level=warning msg="32767:\tfrom all lookup default' ']'" subsys=datapath-loader
level=warning msg="+ ip -4 rule list" subsys=datapath-loader
level=warning msg="+ grep 100" subsys=datapath-loader
level=warning msg="+ grep 'lookup local'" subsys=datapath-loader
level=warning msg="100:\tfrom all lookup local" subsys=datapath-loader
level=warning msg="+ ip -4 rule del from all lookup local pref 0" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="++ ip -4 rule list" subsys=datapath-loader
level=warning msg="++ grep 'lookup local'" subsys=datapath-loader
level=warning msg="++ wc -l" subsys=datapath-loader
level=warning msg="+ '[' 1 -eq 0 ']'" subsys=datapath-loader
level=warning msg="+ '[' '<nil>' '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ setup_proxy_rules" subsys=datapath-loader
level=warning msg="+ from_ingress_rulespec='fwmark 0xA00/0xF00 pref 10 lookup 2005'" subsys=datapath-loader
level=warning msg="+ to_proxy_rulespec='fwmark 0x200/0xF00 pref 9 lookup 2004'" subsys=datapath-loader
level=warning msg="+ '[' 10.42.0.241 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="++ ip -4 rule list" subsys=datapath-loader
level=warning msg="+ '[' -n '9:\tfrom all fwmark 0x200/0xf00 lookup 2004" subsys=datapath-loader
level=warning msg="10:\tfrom all fwmark 0xa00/0xf00 lookup 2005" subsys=datapath-loader
level=warning msg="100:\tfrom all lookup local" subsys=datapath-loader
level=warning msg="32766:\tfrom all lookup main" subsys=datapath-loader
level=warning msg="32767:\tfrom all lookup default' ']'" subsys=datapath-loader
level=warning msg="++ ip -4 rule list fwmark 0x200/0xF00 pref 9 lookup 2004" subsys=datapath-loader
level=warning msg="+ '[' -z '9:\tfrom all fwmark 0x200/0xf00 lookup 2004' ']'" subsys=datapath-loader
level=warning msg="+ '[' false = true ']'" subsys=datapath-loader
level=warning msg="++ ip -4 rule list fwmark 0xA00/0xF00 pref 10 lookup 2005" subsys=datapath-loader
level=warning msg="+ '[' -z '10:\tfrom all fwmark 0xa00/0xf00 lookup 2005' ']'" subsys=datapath-loader
level=warning msg="+ ip route replace table 2004 local 0.0.0.0/0 dev lo" subsys=datapath-loader
level=warning msg="+ '[' false = true ']'" subsys=datapath-loader
level=warning msg="+ ip route replace table 2005 10.42.0.241/32 dev cilium_host" subsys=datapath-loader
level=warning msg="+ ip route replace table 2005 default via 10.42.0.241" subsys=datapath-loader
level=warning msg="+ '[' '<nil>' '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ ip -6 rule del fwmark 0x200/0xF00 pref 9 lookup 2004" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ ip -6 rule del fwmark 0xA00/0xF00 pref 10 lookup 2005" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ '[' tunnel = ipip ']'" subsys=datapath-loader
level=warning msg="+ ip link del cilium_ipip4" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ ip link del cilium_ipip6" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ ip link del cilium_sit" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="+ '[' tunnel = tunnel ']'" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*TUNNEL_MODE.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="+ echo '#define TUNNEL_MODE 1'" subsys=datapath-loader
level=warning msg="+ '[' vxlan '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ ENCAP_DEV=cilium_vxlan" subsys=datapath-loader
level=warning msg="+ ip link show cilium_vxlan" subsys=datapath-loader
level=warning msg="101: cilium_vxlan: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000" subsys=datapath-loader
level=warning msg="    link/ether 2e:9c:ea:48:43:28 brd ff:ff:ff:ff:ff:ff" subsys=datapath-loader
level=warning msg="+ '[' 8472 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ ip -details link show cilium_vxlan" subsys=datapath-loader
level=warning msg="+ grep 'dstport 8472'" subsys=datapath-loader
level=warning msg="    vxlan external id 0 srcport 0 0 dstport 8472 nolearning ttl auto ageing 300 udpcsum noudp6zerocsumtx noudp6zerocsumrx addrgenmode eui64 numtxqueues 1 numrxqueues 1 gso_max_size 65536 gso_max_segs 65535 " subsys=datapath-loader
level=warning msg="+ ip link set cilium_vxlan mtu 1500" subsys=datapath-loader
level=warning msg="+ setup_dev cilium_vxlan" subsys=datapath-loader
level=warning msg="+ local -r NAME=cilium_vxlan" subsys=datapath-loader
level=warning msg="+ ip link set cilium_vxlan up" subsys=datapath-loader
level=warning msg="+ '[' '<nil>' '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ '[' 10.42.0.241 '!=' '<nil>' ']'" subsys=datapath-loader
level=warning msg="+ echo 1" subsys=datapath-loader
level=warning msg="+ echo 0" subsys=datapath-loader
level=warning msg="+ echo 1" subsys=datapath-loader
level=warning msg="+ echo 0" subsys=datapath-loader
level=warning msg="++ cat /sys/class/net/cilium_vxlan/ifindex" subsys=datapath-loader
level=warning msg="+ ENCAP_IDX=101" subsys=datapath-loader
level=warning msg="+ sed -i '/^#.*ENCAP_IFINDEX.*$/d' /var/run/cilium/state/globals/node_config.h" subsys=datapath-loader
level=warning msg="+ echo '#define ENCAP_IFINDEX 101'" subsys=datapath-loader
level=warning msg="+ CALLS_MAP=cilium_calls_overlay_2" subsys=datapath-loader
level=warning msg="+ COPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1'" subsys=datapath-loader
level=warning msg="+ '[' false = true ']'" subsys=datapath-loader
level=warning msg="+ bpf_load cilium_vxlan '-DSECLABEL=2 -DFROM_ENCAP_DEV=1' ingress bpf_overlay.c bpf_overlay.o from-overlay cilium_calls_overlay_2" subsys=datapath-loader
level=warning msg="+ DEV=cilium_vxlan" subsys=datapath-loader
level=warning msg="+ OPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1'" subsys=datapath-loader
level=warning msg="+ WHERE=ingress" subsys=datapath-loader
level=warning msg="+ IN=bpf_overlay.c" subsys=datapath-loader
level=warning msg="+ OUT=bpf_overlay.o" subsys=datapath-loader
level=warning msg="+ SEC=from-overlay" subsys=datapath-loader
level=warning msg="+ CALLS_MAP=cilium_calls_overlay_2" subsys=datapath-loader
level=warning msg="++ ip link show cilium_vxlan" subsys=datapath-loader
level=warning msg="++ grep ether" subsys=datapath-loader
level=warning msg="++ awk '{print $2}'" subsys=datapath-loader
level=warning msg="+ NODE_MAC=2e:9c:ea:48:43:28" subsys=datapath-loader
level=warning msg="++ mac2array 2e:9c:ea:48:43:28" subsys=datapath-loader
level=warning msg="++ echo '{0x2e,0x9c,0xea,0x48,0x43,0x28}'" subsys=datapath-loader
level=warning msg="+ NODE_MAC='{.addr={0x2e,0x9c,0xea,0x48,0x43,0x28}}'" subsys=datapath-loader
level=warning msg="+ OPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DNODE_MAC={.addr={0x2e,0x9c,0xea,0x48,0x43,0x28}} -DCALLS_MAP=cilium_calls_overlay_2'" subsys=datapath-loader
level=warning msg="+ bpf_compile bpf_overlay.c bpf_overlay.o obj '-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DNODE_MAC={.addr={0x2e,0x9c,0xea,0x48,0x43,0x28}} -DCALLS_MAP=cilium_calls_overlay_2'" subsys=datapath-loader
level=warning msg="+ IN=bpf_overlay.c" subsys=datapath-loader
level=warning msg="+ OUT=bpf_overlay.o" subsys=datapath-loader
level=warning msg="+ TYPE=obj" subsys=datapath-loader
level=warning msg="+ EXTRA_OPTS='-DSECLABEL=2 -DFROM_ENCAP_DEV=1 -DNODE_MAC={.addr={0x2e,0x9c,0xea,0x48,0x43,0x28}} -DCALLS_MAP=cilium_calls_overlay_2'" subsys=datapath-loader
level=warning msg="+ clang -O2 -target bpf -std=gnu89 -nostdinc -emit-llvm -g -Wall -Wextra -Werror -Wshadow -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end -Wdeclaration-after-statement -Wimplicit-int-conversion -Wenum-conversion -I. -I/run/cilium/state/globals -I/var/lib/cilium/bpf -I/var/lib/cilium/bpf/include -D__NR_CPUS__=2 -DENABLE_ARP_RESPONDER=1 -DSECLABEL=2 -DFROM_ENCAP_DEV=1 '-DNODE_MAC={.addr={0x2e,0x9c,0xea,0x48,0x43,0x28}}' -DCALLS_MAP=cilium_calls_overlay_2 -c /var/lib/cilium/bpf/bpf_overlay.c -o -" subsys=datapath-loader
level=warning msg="+ llc -march=bpf -mcpu=v3 -mattr=dwarfris -filetype=obj -o bpf_overlay.o" subsys=datapath-loader
level=warning msg="+ tc qdisc replace dev cilium_vxlan clsact" subsys=datapath-loader
level=warning msg="RTNETLINK answers: Operation not supported" subsys=datapath-loader
level=warning msg="+ true" subsys=datapath-loader
level=warning msg="++ tc filter show dev cilium_vxlan ingress" subsys=datapath-loader
level=warning msg="++ grep -v 'pref 1 bpf chain 0 $\\|pref 1 bpf chain 0 handle 0x1'" subsys=datapath-loader
level=warning msg="RTNETLINK answers: Operation not supported" subsys=datapath-loader
level=warning msg="Dump terminated" subsys=datapath-loader
level=warning msg="+ '[' -z '' ']'" subsys=datapath-loader
level=warning msg="+ cilium bpf migrate-maps -s bpf_overlay.o" subsys=datapath-loader
level=warning msg="+ tc filter replace dev cilium_vxlan ingress prio 1 handle 1 bpf da obj bpf_overlay.o sec from-overlay" subsys=datapath-loader
level=warning msg="RTNETLINK answers: Operation not supported" subsys=datapath-loader
level=warning msg="We have an error talking to the kernel" subsys=datapath-loader
level=warning msg="+ cilium bpf migrate-maps -e bpf_overlay.o -r 1" subsys=datapath-loader
level=warning msg="+ return 1" subsys=datapath-loader
level=fatal msg="Error while creating daemon" error="error while initializing daemon: failed while reinitializing datapath: exit status 1" subsys=daemon
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by joestringer** [comment]
These logs seem to indicate that the kernel does not have the required features enabled to run Cilium:

```
level=warning msg="+ tc qdisc replace dev cilium_vxlan clsact" subsys=datapath-loader
level=warning msg="RTNETLINK answers: Operation not supported" subsys=datapath-loader
```

Is this just a default Ubuntu 5.15 kernel?

**Comment by Hugo-Azix** [comment]
Hello @joestringer,

We use proxmox on which we generate virtual machines thanks to a cloud-init, we use this image: 

https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64-disk-kvm.img

And our Kernel Version is : Linux test 5.15.0-1025-kvm 30-Ubuntu SMP Thu Dec 1 22:50:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

Do you have more help/ideas for me? or I just have to try to change the OS version

**Comment by joestringer** [comment]
I am not familiar with the details of that image, but the kernel config requirements are documented in the below link. You may be able to compare this vs. the `/boot/config*` files in your environment: 

https://docs.cilium.io/en/stable/operations/system_requirements/#linux-kernel

**Comment by Hugo-Azix** [comment]
@joestringer 
Indeed, I am missing some lines in the config file. I'll look into it tomorrow but
I'm sorry to ask this question because I'm a beginner, I have to compare and i can change the configuration file with the requirements? or I have to find an image that matches with the requirements?

thank you for your help

**Comment by joestringer** [comment]
You will need to find an image that matches the requirements.

**Comment by bakito** [comment]
Hi 

I run into the same issue.
The ubuntu cloud image does not provide the required kernel modules.

I Swi

> You will need to find an image that matches the requirements.

I run into the same issue with the ubuntu cloud image, which does not provide the required kernel modules.

Switching to to a debian cloud image https://cloud.debian.org/images/cloud/bookworm/latest/debian-12-generic-amd64.qcow2 solved the issue.
